libraries/3rdparty/CMakeLists.txt

```diff
@@ -27,3 +27,22 @@ if(${BUILD_TESTS})
 
     target_link_libraries(cmock unity)
 endif()
+
+add_library(tinycbor STATIC
```

Comment:
It may not be necessary to build this library as a static archive as we provide the option of building all the libraries as shared library files (with `*.so` extension)..